CMakeLists.txt

```diff
@@ -20,6 +20,29 @@ endif ()
 project(remill)
 cmake_minimum_required(VERSION 3.2)
 
+if(EMSCRIPTEN)
+  set(CMAKE_BUILD_TYPE Release)
+  set(gflags_DIR "$ENV{REPOS}/gflags/build")
+  set(glog_DIR "$ENV{REPOS}/glog/build")
+  set(LLVM_DIR "$ENV{REPOS}/llvm-project/llvm/build/lib/cmake/llvm/")
+  set(remill_DIR "$ENV{REPOS}/remill/build/")
+
+  # We want to manually invoke main for Lift instead of Emscripten calling it for us
+  # So we disable INVOKE_RUN, export callMain (and FS for files).
+  set(CMAKE_MODULE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/web/src")
+  set(CMAKE_CXX_FLAGS "\
+    ${CMAKE_CXX_FLAGS} \
+    $ENV{EM_CXX_FLAGS} \
```

Comment:
Can this setting be moved away from the environment variable?

Example, in cmake/settings.cmake
```
set(REMILL_EMSCRIPTEN_CXXFLAGS "default_value" CACHE STRING "Semicolon separated list of flags")
```

This will make the flag list show up as a configurable setting (i.e.: make edit_cache)